src/test/java/org/sagebionetworks/bridge/TestSurvey.java

```diff
@@ -62,10 +62,11 @@ public static SurveyQuestion selectBy(Survey survey, DataType type) {
             Image great = new Image("http://great.svg", 600, 300);
             MultiValueConstraints mvc = new MultiValueConstraints(DataType.INTEGER);
             List<SurveyQuestionOption> options = Lists.newArrayList(
-                    new SurveyQuestionOption("Terrible", null, "1", terrible),
-                    new SurveyQuestionOption("Poor", null, "2", poor), new SurveyQuestionOption("OK", null, "3", ok),
-                    new SurveyQuestionOption("Good", null, "4", good),
-                    new SurveyQuestionOption("Great", null, "5", great));
+                    new SurveyQuestionOption("Terrible", null, "1", terrible, null),
+                    new SurveyQuestionOption("Poor", null, "2", poor, null), 
+                    new SurveyQuestionOption("OK", null, "3", ok, null),
+                    new SurveyQuestionOption("Good", null, "4", good, null),
+                    new SurveyQuestionOption("Great", null, "5", great, null));
```

Comment:
Do we expect to add more features to SurveyQuestionOption over time? If so, this would cause a lot of churn as we'd have to update a lot of constructor calls. Is it worth converting to a Builder pattern at this time?

Reply:
This is the first addition in a few years and I don't have any further requests to add to this. I don't think it's worth creating a builder.